scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -461,13 +171,56 @@ task CheckForDuplicateData {
   }
 }
 
+task CreateLoadStatusTempTable {
```

Comment:
is it worth is to import the CreateTable.wdl and call it to create this temp table?